### DISCUSSION TITLE
Alert text should be copiable to the clipboard.

### DISCUSSION THREAD

**Discussion by theraot** [godotengine/godot-proposals#7183] [Editor]
When I get an alert in the editor (which I believe is an `AcceptDialog`), such as this one:

![image](https://github.com/godotengine/godot/assets/1212509/7ea60403-4bc1-4332-961e-3b9c36bf9216)

I want to press the keyboard shortcut to copy (Ctrl + C) while the alert window has keyboard focus, and I want that to result in the text of the alert being copied to the clipboard.

---

For those who don't know, this is a feature of message boxes in Windows. For example if I use this code:

```
OS.alert("AAA")
```

*Which way much easier to use than `AcceptDialog`, but I digress...*

I get a native message box, which in Windows (in Spanish) looks like this:

![image](https://github.com/godotengine/godot-proposals/assets/1212509/b77b5b85-cb77-48b1-9f4c-4760dcf96d1d)

Which I can copy and paste, giving me this:

```
---------------------------
Alert!
---------------------------
AAA
---------------------------
Aceptar   
---------------------------
```

---

To be fair, I suppose that changing the alert to allow selecting the text, or adding a button to copy the text to the clipboard might make it easier for discoverability. Although, personally I'm not asking for the discoverability.

**Comment by Calinou** [comment]
It was considered to make dialog text use RichTextLabel at some point (so selection could be enabled), but there were some blockers to doing this.
